tests/Common/TokensTest.php

```diff
@@ -1348,8 +1348,7 @@ public function limitedTokenOptionsData()
     {
         return [
             'minimal configuration' => [
-                (new TokenCreateOptions())
-                    ->setExpiresIn(60 * 5),
+                (new TokenCreateOptions()),
```

Comment:
Nevím, jestli mám něco špatně, ale tohle se tváří, že je změna v testu. Je možné, že se to nově vytváří nějak jinak a že ten token nemá expiraci, když ji nezadáš? Neměl limited token muset mít expiraci? Nebo v čem je tam problém? 

![image](https://user-images.githubusercontent.com/642928/189141070-0b298b75-4dfe-4c95-82af-00844b8c4793.png)

Reply:
ja mel testy zeleny... :-/ zkusim

Reply:
Aha, tak to vypada, ze jsem zkousel nejspis jen ten druhej test 🤦 
Ted jsem opravil i ten prvni test - nove je expirace optional u normal i limited tokenu.